dataset_builders/pie/argmicro/README.md

```diff
@@ -0,0 +1,31 @@
+# PIE Dataset Card for "ArgMicro"
+
+This is a [PyTorch-IE](https://github.com/ChristophAlt/pytorch-ie) wrapper for the
+[ArgMicro Huggingface dataset loading script](https://huggingface.co/datasets/DFKI-SLT/argmicro).
+
+## Data Schema
+
+The document type for this dataset is `ArgMicroDocument` which defines the following data fields:
+
+- `id` (str)
+- `text` (str)
+- `topic_id` (str, optional)
+- `stance` (str, optional)
+- `metadata` (dictionary, optional)
+
+and the following annotation layers:
+
+- `edus` (annotation type: `Span`, target: `text`)
+- `adus` (annotation type: `LabeledAnnotationCollection`, target: `edus`)
+- `relations` (annotation type: `MultiRelation`, target: `adus`)
```

Comment:
both of these types are a bit special, can you add a subitem below each of them that shortly explains what they are?

Reply:
edit in 8e97ed3

Reply:
hmm this still is not so clear to me. I restructured this a bit and added some TODOs, see https://github.com/ArneBinder/pie-datasets/pull/56/commits/3723f60d2e66a35374fedda810fb10eb5528255b. Please look into them.

Reply:
made further changes in 2a9c40a, please have a look